flags.css

```diff
@@ -1,8 +1,56 @@
 /* Indonesia */
+.indonesiaFlag {
+    border: 2px solid;
+    height: 200px;
+    width: 300px;
+}
+.indonesiaStrip1 {
+    background-color: red;
+    height: 100px;
+}
 
 /* Colombia */
-
+.columbiaFlag {
+    margin-top: 50px;
+    border: 2px solid;
+    height: 25vh;
+    width: 300px;
+}
+.columbiaStrip1 {
+    background-color: gold;
+    height: 50%;
+}
+.columbiaStrip2,
+.columbiaStrip3 {
```

Comment:
you can often think of 25% height or width as 50% of the previous if the previous was 50%.  Can be very handy for class reuse.  Check that out here.

https://jsbin.com/rabuguc/2/edit?html,css,output